jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpCompliance.java

```diff
@@ -402,9 +402,6 @@ public static void checkHttpCompliance(MetaData.Request request, HttpCompliance
                     for (String hostValue: hostValues)
                         if (StringUtil.isBlank(hostValue))
                             assertAllowed(Violation.UNSAFE_HOST_HEADER, mode, listener);
-                    String authority = request.getHttpURI().getHost();
-                    if (StringUtil.isBlank(authority))
-                        assertAllowed(Violation.UNSAFE_HOST_HEADER, mode, listener);
```

Comment:
This is just an unsafe compliance violation, id doesn't fail the header if you allow that specific violation.  Please restore these lines of code, if not here, somewhere else.

Reply:
Had conversation with @sbordet - i'm sold on the new future authority check PR